src/screens/courses/CourseList.tsx

```diff
@@ -17,8 +16,12 @@ import { PINK, YELLOW } from '../../styles/colors';
 import NextStepCell from '../../components/steps/NextStepCell';
 import { FIRA_SANS_BOLD } from '../../styles/fonts';
 
+interface CourseListScreenProps {
+  navigation: object,
+}
+
 const formatDataForNextSteps = (courses) => {
-  const futureSlots = [];
+  const futureSlots = new Array(0);
```

Comment:
pourquoi ce changement ?

Reply:
'Type 'any' is not assignable to type 'never'.ts(2322)'
Typescript veut cette précision. Une autre résolution serait de typer l'array :
`futureSlots : futureSlotType[] = [];`

Reply:
C'est ce que je ferai lors du refacto d'ailleurs

Reply:
je n'ai pas compris...